[esx]/esx_context/main.lua

```diff
@@ -74,11 +75,11 @@ end)
 -- NUI Callbacks
 -- [ closed | selected | changed ]
 
-RegisterNUICallback("closed",function()
-	if not activeMenu then
-		return
+RegisterNUICallback("closed",function(data,cb)

```

Comment:
var data unnecessary no ?

Reply:
Not necessary but afaik you need to put data there so that you can pass callback as the second param.